demisto_sdk/commands/validate/validators/BC_validators/BC105_id_changed.py

```diff
@@ -0,0 +1,121 @@
+
+from typing import Iterable, List, Union, cast
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.assets_modeling_rule import (
+    AssetsModelingRule,
+)
+from demisto_sdk.commands.content_graph.objects.classifier import Classifier
+from demisto_sdk.commands.content_graph.objects.correlation_rule import CorrelationRule
+from demisto_sdk.commands.content_graph.objects.dashboard import Dashboard
+from demisto_sdk.commands.content_graph.objects.generic_definition import (
+    GenericDefinition,
+)
+from demisto_sdk.commands.content_graph.objects.generic_field import GenericField
+from demisto_sdk.commands.content_graph.objects.generic_module import GenericModule
+from demisto_sdk.commands.content_graph.objects.generic_type import GenericType
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.content_graph.objects.incident_type import IncidentType
+from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.job import Job
+from demisto_sdk.commands.content_graph.objects.layout import Layout
+from demisto_sdk.commands.content_graph.objects.layout_rule import LayoutRule
+from demisto_sdk.commands.content_graph.objects.list import List as LIST
+from demisto_sdk.commands.content_graph.objects.mapper import Mapper
+from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule
+from demisto_sdk.commands.content_graph.objects.parsing_rule import ParsingRule
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.pre_process_rule import PreProcessRule
+from demisto_sdk.commands.content_graph.objects.report import Report
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.objects.trigger import Trigger
+from demisto_sdk.commands.content_graph.objects.widget import Widget
+from demisto_sdk.commands.content_graph.objects.wizard import Wizard
+from demisto_sdk.commands.content_graph.objects.xsiam_dashboard import XSIAMDashboard
+from demisto_sdk.commands.content_graph.objects.xsiam_report import XSIAMReport
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    LIST,
+    Mapper,
+    Classifier,
+    Widget,
+    Integration,
+    Dashboard,
+    IncidentType,
+    Script,
+    Playbook,
+    Report,
+    Wizard,
+    Job,
+    Layout,
+    PreProcessRule,
+    CorrelationRule,
+    ParsingRule,
+    ModelingRule,
+    XSIAMDashboard,
+    Trigger,
+    XSIAMReport,
+    IncidentField,
+    IndicatorField,
+    AssetsModelingRule,
+    LayoutRule,
+]
```

Comment:
@YuvHayun Do all content items have the field **object_id** configured?

Reply:
Yap.